src/main/webapp/app/course/tutorial-groups/tutorial-groups-management/tutorial-groups-checklist/tutorial-groups-checklist.component.html

```diff
@@ -1,3 +1,4 @@
+<jhi-course-management-tab-bar></jhi-course-management-tab-bar>
```

Comment:
Can this somehow be automatically inserted at a general location? Now it's easy to miss pages where this component should be used and also new developers must remember to add this component on newly added pages.

Reply:
The visibility of the buttons is restricted only to the pages to which they directly link to, except for Communications, which links to a page in course-overview rather than course-management. As a result, adding this component to the top of a new page is necessary only if there is a corresponding button added to the course-management-tab-bar component. 

In my opinion, new pages with this component should also be linked to by the buttons to ensure a seamless user experience. It may negatively impact the user experience if they can navigate away from a page using the course-management-tab-bar component but not back to it.

Reply:
> It may negatively impact the user experience if they can navigate away from a page using the course-management-tab-bar component but not back to it.

Totally agree with you on that. But that's exactly why I thought we should look out for an alternative. You could easily add a new button but forget to add the tab bar component, leading to an inconsistent user experience.

Would it be possible to inject that component for all pages of the form /course-management/id/page? This would also not affect the communication button since this links to /course/id/discussion.

Reply:
That's a reasonable idea. Rather than injecting anything, we could create a wrapper component that wraps all course management pages using a router outlet. We could place the menu bar at the top of this component. Shouldn't be too hard to implement

Reply:
Thank you both for providing valuable feedback.

After reviewing the course overview, I have noticed that the suggested solution by @bassner is very similar to the one implemented there. Therefore, I will revise my PR to include a router outlet and create a new wrapper component for course management, following the example set by the course-overview.component.
Additionally, I will attempt to integrate the header component and adjust the button styles to be consistent with the course overview and clearly indicate the user's current page 🚀